recipe/meta.yaml

```diff
@@ -60,6 +60,8 @@ outputs:
         - sqlite  # [not win]
         - cuda-cudart-dev                           # [(cuda_compiler_version or "").startswith("12")]
         - libcublas-dev                             # [(cuda_compiler_version or "").startswith("12")]
+        - libcurand-dev                             # [(cuda_compiler_version or "").startswith("12")]
+        - libcusolver-dev                           # [(cuda_compiler_version or "").startswith("12")]
         - cuda-nvcc-tools                           # [(cuda_compiler_version or "").startswith("12")]
         - cuda-nvcc-impl                            # [(cuda_compiler_version or "").startswith("12")]
```

Comment:
Could we drop these from `host`? Think this is confusing CMake on Windows

Also please let me know any context as to why they were added

Reply:
Sure thing. Unfortunately still the same issue :(. In https://github.com/conda-forge/pycolmap-feedstock/pull/16 I went back to a setuptools build, that fixed it somehow. I think the correct flags are still not being passed.

/cc @carterbox